### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by rafidude**
Minimal style added...
https://gist.github.com/rafidude/8d0eddd2b7c4664bf7e4

### CODE REVIEW

**Comment by andrewmd5** [comment]
Duplicate of #66

File: index.html [hunk 1]
```diff
@@ -22,6 +22,17 @@
...
+    body {
+	    margin:40px auto;
+	    max-width:650px;
+	    line-height:1.6;
+	    font-size:18px;
+	    color:#444;
+	    padding:0 10px
+    }
+    h1,h2,h3{line-height:1.2}
+    </style>
```
Comment (nightpool) [inline index.html:35]: any reason why you added a separate style tag? Just curious

Comment (rafidude) [inline index.html:35]: Wanted to leave the existing style in tact. The existing ones override the styles I added. Easier to maintain as well.

**Comment by ThomasLomas** [comment]
Read http://motherfuckingwebsite.com/ - it's in the README file.

**Comment by nightpool** [comment]
@ThomasLomas right—which says you should have _lightweight_ websites. not stupid looking ones. read the last paragraph, under "satire".

**Comment by ThomasLomas** [comment]
@nightpool My point is that the website is completely functional as is. As an engineer utilising this I expect functionality and not exactly aesthetics.

Increasing line height for readability I get. Aligning centre and all that not so much.

**Comment by nightpool** [comment]
line width is just as important, if not more important, then line height
for readability.

**Comment by ThomasLomas** [comment]
@nightpool in that case 650px in my opinion is definitely not the number to pick (well 630px after padding). 

If we're going back to centre aligned layouts of fixed size, then perhaps 960px would be a better starting point.

**Comment by diafygi** [comment]
Thanks for the PR, but will likely merge in 752dd56 _if and only if_ I decide I want to. The website works now, so this is just window dressing.

**Pull request closed without merging**